.github/workflows/changelog.yaml

```diff
@@ -28,7 +28,7 @@ jobs:
       - run: ./hack/changelog.sh > CHANGELOG.md
       - uses: peter-evans/create-pull-request@a4f52f8033a6168103c2538976c07b467e8163bc # v6.0.1
         with:
-          title: 'docs: updated CHANGELOG.md'
-          commit-message: 'docs: updated CHANGELOG.md'
+          title: 'docs: update CHANGELOG.md for ${{ github.ref_name }}'
+          commit-message: 'docs: update CHANGELOG.md for ${{ github.ref_name }}'
```

Comment:
I guess there's no good way to test this, huh? In that case, I just wanted to make sure there was no risk of `${{ github.ref_name }}` not getting evaluated or not getting evaluated correctly, but I do see it listed [here](https://docs.github.com/en/actions/learn-github-actions/contexts) as "The short ref name of the branch or tag that triggered the workflow run. This value matches the branch or tag name shown on GitHub." which in this case should be the tag. So, I think I'm good.

Reply:
Other than making a fork and triggering actions (which is _very_ tedious) unfortunately not. 

[`act`](https://github.com/nektos/act) is the closest equivalent for local running, and it doesn't quite support everything either; not sure if it would work for this one.